goodreads.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict
+
+from .api import search_goodreads
+
+def get_original_date(self, asin: str, authors: str, title: str) -> Dict:
+    api_key = self.config['goodreads_apikey']
+    goodreads_response = search_goodreads(api_key, asin)
+    totalresults = goodreads_get_total_result(goodreads_response)
+
+    if totalresults == 0:
+        #search with author and title
+        #set asin to none to determine that we are querying on author/title
```

Comment:
Where is asin being set to None?

Reply:
The wording on that comment isn't great. In the first call to `search_goodreads` above, I use ASIN as the only search criteria. When no results are found, I call `search_goodreads` again but without ASIN, just author and title. 

I'll remove that comment.